tests/filters/fconnection.py

```diff
@@ -0,0 +1,226 @@
+import logging
+
```

Comment:
Caution, import os is missing. Used line 29.

```suggestion
import os

```